cli/cli-client/src/main/java/org/infinispan/cli/commands/kubernetes/Kube.java

```diff
@@ -0,0 +1,107 @@
+package org.infinispan.cli.commands.kubernetes;
+
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+import org.aesh.command.Command;
+import org.aesh.command.CommandResult;
+import org.aesh.command.GroupCommandDefinition;
+import org.aesh.command.invocation.CommandInvocation;
+import org.infinispan.cli.commands.Version;
+import org.infinispan.cli.logging.Messages;
+import org.yaml.snakeyaml.Yaml;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.client.KubernetesClient;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.0
+ **/
+@GroupCommandDefinition(
+      name = Kube.CMD,
+      description = "",
+      groupCommands = {
+            Install.class,
+            Interactive.class,
+            Create.class,
+            Ls.class,
+            Remove.class,
+            Version.class
+      })
+public class Kube implements Command {
+   public static final String CMD = "kube";
+   public static final String DEFAULT_NAMESPACE = "default";
+   public static final String DEFAULT_SERVICE_NAME = "infinispan";
```

Comment:
Service is a bit overload. Kubernetes Service and Infinispan service are confusing. Maybe call `DEFAULT_INFINISPAN_NAME` or `DEFAULT_CLUSTER_NAME`?